src/components/Header.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import '../style/Header.css';
+// import { Link } from 'react-router-dom';
```

Comment:
- [ ] Please remove commented code to write clean code.
- [ ] For now please only have book.js in component directory. There should no other file.